files/default/test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+coverage
```

Comment:
Was documented in the readme.md, and is a good step before travis integration. IMHO it is not hurting.